### ISSUE TITLE
"Implicit braces are forbidden" triggered with function call in class body

### ISSUE THREAD

**Issue by delaaxe** [clutchski/coffeelint#68]
this valid coffeescript triggers the wrong error when calling a function in class body:

```
class A

  @configure(1, 2, 3)

  constructor: ->
```

pattern used all over the place by the spine framework

edit: that is, with the "no_implicit_braces" set to "error" of course, and which have nothing to do with the preceding example
